BrainPortal/app/controllers/groups_controller.rb

```diff
@@ -151,8 +151,11 @@ def update #:nodoc:
       end
     end
 
+    unless (current_user.available_users.map{ |u| u.id } | @group.user_ids).include?(params[:group][:creator_id].to_i )
```

Comment:
OK, it's a workaround that will do for the moment; the logging helper doesn't know how to link the attribute `creator_id` to the `User` model (even though Rails know about it) because it does a simple parsing of the attribute name and tries to find a class `Creator` instead. The fix would be to make it understand some Rails associations in `BrainPortal/lib/act_rec_log.rb` at the line that says

```ruby
model = Regexp.last_match[1].classify.constantize rescue nil
```